emails/views.py

```diff
@@ -443,6 +443,9 @@ def _sns_message(message_json):
             message_json
         )
     except ClientError as e:
+        if e.response['Error'].get('Code', '') == 'NoSuchKey':
+            logger.error('s3_object_does_not_exist', extra=e.response['Error'])
```

Comment:
maybe we should use `logger.exception` so that a traceback is included, and we know which line raised the error. But I'm not sure if that will be logged cleanly, or as an ugly multi-line entry. So, let's stay on `logger.error`.

Reply:
We can do this as a follow-up.